src/exercise.rs

```diff
@@ -58,6 +60,24 @@ impl Exercise {
                 .args(&["--test", self.path.to_str().unwrap(), "-o", &temp_file()])
                 .args(RUSTC_COLOR_ARGS)
                 .output(),
+            Mode::Clippy => {
+                let cargo_toml = format!(
+                    r#"[package]
+name = "{}"
+version = "0.0.1"
+edition = "2018"
+[[bin]]
+name = "{}"
+path = "{}.rs""#,
+                    self.name, self.name, self.name
+                );
+                fs::write(CLIPPY_CARGO_TOML_PATH, cargo_toml)
+                    .expect("Failed to write 📎 Clippy 📎 Cargo.toml file.");
+                Command::new("cargo")
+                    .args(&["clippy", "--manifest-path", CLIPPY_CARGO_TOML_PATH])
```

Comment:
This would imply that you have to have Clippy installed, right? As far as I know, Clippy isn't added as a default Rustup component, so you'd have to mention somewhere that you need to explicitly do `rustup component add clippy`

Reply:
You're right.
I added the Clippy installation to the install scripts. I assume this should be good then?